test/project1/source/colors/templateStrings/color-alpha-mixed-values.bs

```diff
@@ -0,0 +1,11 @@
+sub init()
+    color = `0xfffff0FF`
```

Comment:
You need more cases, like empty strings, complex templates with variables (at the beginning, end...).